index.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once 'Autor.php';
```

Comment:
```suggestion
require_once "Autor.php";
```
Trata de usar un solo tipo de estilo de comillas, ya sean simples o dobles